src/ui/components/EntryIcon/EntryIcon.tsx

```diff
@@ -153,7 +154,12 @@ export const getEntryIconData = ({scope, type}: EntryData) => {
         if (scope === 'widget' && !Utils.isEnabledFeature(Feature.EntryMenuEditor)) {
             typeKey = '';
         }
-        const icon = typeToIcon[typeKey];
+        const {getAdditionalEntryIcons} = registry.common.functions.getAll();
+        const typeToIconExtended = {
+            ...typeToIcon,
+            ...getAdditionalEntryIcons(),
+        };
+        const icon = typeToIconExtended[typeKey];
```

Comment:
Maybe you should use `getIconDataById` from `registry.common.functions` here? And do not create an additional extension